src/components/Chat/InputMessage.js

```diff
@@ -130,7 +130,7 @@ const styles = StyleSheet.create({
     paddingLeft: 6
   },
   containerInput: {
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: colors.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value has been changed. Ensure that the new color (`colors.lightgrey`) is equivalent to the old one (`COLORS.lightgrey`) in terms of user experience and accessibility.
```diff
- backgroundColor: COLORS.lightgrey,
+ backgroundColor: colors.lightgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->